pages/[lng]/payment_method/index.tsx

```diff
@@ -129,26 +129,27 @@ const PaymentMethods: FC<any> = ({
   const linksBreadcrumb = [`${i18n.t("header.home")}`, i18n.t("placeOrder.checkOrder")]
```

Comment:
```suggestion
  const linksBreadcrumb = [i18n.t("header.home"), i18n.t("placeOrder.checkOrder")]
```

giniin boleh pacc? gatel saia liatnya